Improve .cspoj file and Directory.Build.props file

Abrynos opened the pull request:
## Changes

- Copyright in `Directory.Build.Props` is updated. A plugin that is created in the year 2022 should not have a copyright notice starting in the year 2021
- `JustArchiNET.Madness` is now included the same way it is done over in ASFs main repository.
- Removed license expression. A plugin maintainer should be able to decide themselves, which licence their project is using.

Abrynos committed:
Update Directory.Build.props

Abrynos committed:
Update MyAwesomePlugin.csproj

Abrynos committed:
Update Directory.Build.props

Abrynos committed:
Update Directory.Build.props

JustArchi commented:
> Removed license expression. A plugin maintainer should be able to decide themselves, which licence their project is using.

He can, and we still want to provide default for him. I'd like you to revert this, other stuff makes sense.

JustArchi reviewed (changes requested):
As per above

Abrynos committed:
Update Directory.Build.props

I personally am a big fan of the GPL-2.0 licence, but I don't want to start that discussion again with Archi

JustArchi commented:
Thanks as usual! :trophy:
